network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -1,9 +1,9 @@
 {% extends "./bg_base.html" %}
 
-{% load bg_selector_tags env cloudinary l10n i18n static %}
+{% load bg_selector_tags env cloudinary l10n i18n static wagtailimages_tags %}
 
 {% block head_extra %}
-  <meta property="og:title" content="{% blocktrans context "This can be localized. This is a reference to the “*batteries not included” mention on toys." %}privacy not included - {{ product.name }}{% endblocktrans %}" />
+  <meta property="og:title" content="{% blocktrans context "This can be localized. This is a reference to the “*batteries not included” mention on toys." %}privacy not included - {{ product.title }}{% endblocktrans %}" />
```

Comment:
This is a Wagtail Page now. We don't need the `.name` property. We're leveraging the Page's built in `.title` property